sensor_msgs/Image color
float32[] depth
sensor_msgs/CameraInfo cameraInfo
float32[] extRotationMatrix
float32[] extTranslationVector
float64 depthScale
